packages/kbn-securitysolution-io-ts-list-types/src/common/exception_list/index.ts

```diff
@@ -20,6 +20,7 @@ export type ExceptionListTypeOrUndefined = t.TypeOf<typeof exceptionListTypeOrUn
 export enum ExceptionListTypeEnum {
   DETECTION = 'detection',
   ENDPOINT = 'endpoint',
+  ENDPOINT_TRUSTED_APPS = 'endpoint',
```

Comment:
I'm keeping the type as it is defined today. We'll change it in the future and provide migration if necessary.

Reply:
Yes, that is correct. It is currently using this type for Trusted apps. When we first implemented Trusted Apps I remember trying to change the type, but that trickled up all the way to the Detections code in the UI and we did not have time to refactor it. I think there is an Issue open to move it to its own type, which will likely also need migration.